crates/solver/src/solver/oneinch_solver.rs

```diff
@@ -161,7 +201,17 @@ impl SingleOrderSolving for OneInchSolver {
             .protocol_cache
             .get_allowed_protocols(&self.disabled_protocols, self.client.as_ref())
             .await?;
-        self.settle_order_with_protocols(order, protocols).await
+        let slippage = Self::compute_max_slippage(
+            auction.external_prices.price(&order.buy_token).expect(
+                "auction should only contain orders where prices \
+                    for buy_token and sell_token are known",
+            ),
+            &order.buy_amount,
+            self.oneinch_slippage_bps,
+            &self.max_slippage_in_wei,
+        )?;
+        self.settle_order_with_protocols_and_slippage(order, protocols, slippage)
+            .await
```

Comment:
We should probably debug log the computed slippage. It is likely that this already happens in another code path for example because we log the url, just wanted to mention.